learn-CSS-by-Building-a-blog/blog56.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
```

Comment:
This seems to be the same step as the one before. It can be updated in the next draft.